src/components/ContactDetails.vue

```diff
@@ -570,7 +624,35 @@ export default {
 
 			return ''
 		},
+		profilePageLink() {
+			// just for testing this is not fully correct
+			if (this.profileEnabled) {
+				return generateUrl('/u/{userId}', { userId: getCurrentUser().uid })
+			}
+			return null
+		},
```

Comment:
analyze the result of this computed property when viewing someone else's contact

Reply:
yes, this is not correct, i already tested it and i always get my uid for every user i create from my account, thats why i left the comment //testing